### DISCUSSION TITLE
Add description field

### DISCUSSION THREAD

**Discussion by yassirnajmaoui** [davish/obsidian-full-calendar#397] [Feature Requests]
It is very useful to have a **description** field in the events that would be displayed in the interface.

I think it would be useful for, for example, adding addresses to events or some extra info.

**Comment by chasoft** [comment]
Hello, kindly implement this change.. this would help speed up when working with the calendar. I often take note and i must every time click "Open note".. it takes time. Please consider adding a textarea as a quick way to add small note (instead of clicking "Open note").

Thank you!

![image](https://user-images.githubusercontent.com/7332422/194685570-8e20b13e-ff6d-44b2-81d5-19399ec5c554.png)
